src/api/app/controllers/issue_trackers_controller.rb

```diff
@@ -46,7 +46,7 @@ def create
     end
   end
 
-  # PUT /issue_trackers/bnc
+  # PUT /issue_trackers/<id>
```

Comment:
This is a name, not an id. Same for the other change in this file.

```suggestion
  # PUT /issue_trackers/<name>
```